resources/META-INF/plugin.xml

```diff
@@ -61,7 +61,7 @@
     ]]></description>
 
   <change-notes><![CDATA[
-      1.5.1 Improved reporting of execution failures. (Contribution by @firewave)<br/>
+      Parse --xml output instead of text output. (Contribution by @firewave)<br/>
```

Comment:
There's no need to have the version in the change note looking at the version history on the plugin page - see https://plugins.jetbrains.com/plugin/8143-cppcheck/versions/stable/102059